src/js/components/Form/stories/Controlled.js

```diff
@@ -47,8 +47,9 @@ export const Controlled = () => {
           <FormField label="Name" name="name">
             <TextInput name="name" suggestions={suggestions} />
           </FormField>
-          <FormField label="Email" name="email" required>
+          <FormField label="Email" htmlFor="email" name="email" required>
```

Comment:
Do we need to have the `htmlFor` prop for 'required' to get read out correctly? I noticed required is being read out for this story but not for other Form stories

Reply:
Yes, but FormField always should have `htmlFor` and `id` on input in order to associate the label with the input, regardless of it's required or not. (you'll see not associated label flagged in the accessibility panel for all form stories). It's something we should fix in our stories but I think should be handled in a separate PR.

Reply:
Sounds good. Can we open up a separate issue to change the stories to use `htmlFor` and `id`?

Reply:
https://github.com/grommet/grommet/issues/6542